addons/main/XEH_postServerInit.sqf

```diff
@@ -41,4 +41,15 @@ if (GVAR(crashHelfer)) then {[true] spawn FUNC(crashHelferServer)};
     1800
 ] call CBA_fnc_addPerFrameHandler;
 
+// Admin Logging
+[
+    "TB_informAdminsAndZeus",
+    {
+        if (isServer && {!isNil QGVAR(loggingExtension)} && {GVAR(loggingExtension)}) then
+        {
+            "TBModExtension" callExtension ["logger", ["adminlog", "ADMIN", if (_this isEqualType []) then {format _this} else {_this}]];
```

Comment:
Sicher das in der Mission kein systemchat logs mehr ausgegeben werden sollen?

Reply:
wtf danke für den Hinweis.. das sollte nicht sein... warum habe ich das aus client weggelöscht obwohl es auf server scon gab usw kA... repariere ich dann ^^

Reply:
Ist noch drin, ist mir aufgefallen:
https://github.com/TacticalBaconDevs/TBMod/blob/436e4d6ec0c4870d0866137be402c5a12599e307/addons/main/functions/fnc_miscStuff.sqf#L146-L156